python/ray/autoscaler/aws/cloudwatch/example-cloudwatch-dashboard-config.json

```diff
@@ -0,0 +1,331 @@
+               "processes_total",
+               "InstanceId",
+               "{instance_id}",
+               {
+                  "label":"Total",
+                  "color":"#69cbe2"
+               }
+            ],
+            [
+               ".",
+               "processes_total_threads",
+               ".",
+               ".",
+               {
+                  "label":"Total Threads",
+                  "color":"#f1b3aa"
+               }
+            ]
+         ],
+         "region":"{region}",
+         "stat":"Average",
+         "period":60,
+         "title":"Process Count"
+      }
+   },
+   {
+      "type":"metric",
+      "x":0,
+      "y":18,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "stacked":false,
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
+               "disk_used_percent",
+               "path",
+               "/dev",
+               "InstanceId",
+               "{instance_id}",
+               "device",
+               "udev",
+               "fstype",
+               "devtmpfs"
+            ],
+            [
+               "...",
+               "/run",
+               ".",
+               ".",
+               ".",
+               "tmpfs",
+               ".",
+               "tmpfs"
+            ],
+            [
+               "...",
+               "/",
+               ".",
+               ".",
+               ".",
+               "nvme0n1p1",
+               ".",
+               "ext4"
+            ],
+            [
+               "...",
+               "/dev/shm",
+               ".",
+               ".",
+               ".",
+               "tmpfs",
+               ".",
+               "tmpfs"
+            ],
+            [
+               "...",
+               "/run/lock",
+               ".",
+               ".",
+               ".",
+               ".",
+               ".",
+               "."
+            ],
+            [
+               "...",
+               "/sys/fs/cgroup",
+               ".",
+               ".",
+               ".",
+               ".",
+               ".",
+               "."
+            ],
+            [
+               "...",
+               "/snap/core/8689",
+               ".",
+               ".",
+               ".",
+               "loop1",
+               ".",
+               "squashfs"
+            ],
+            [
+               "...",
+               "/snap/amazon-ssm-agent/1480",
+               ".",
+               ".",
+               ".",
+               "loop3",
+               ".",
+               "."
+            ]
+         ],
+         "region":"{region}",
+         "title":"Disk used"
+      }
+   },
+   {
+      "type":"metric",
+      "x":12,
+      "y":18,
+      "width":12,
+      "height":6,
+      "properties":{
+         "view":"timeSeries",
+         "metrics":[
+            [
+               "{cluster_name}-ray-CWAgent",
```

Comment:
I think we should convert this to an average and a maximum cluster-level metric using search expressions (similar to what we're doing with `{cluster_name}-ray-prometheus` metrics below). Same for `disk_used_percent` and `cpu_usage_system`).